how-it-works.html

```diff
@@ -0,0 +1,254 @@
+                environment,
+                Ocopea needs the ability to make copies of each individual component configuration and data:
+                <br/>
+            </p>
+            <ul class="list">
+                <li>
+                    The hackathon docker image version that is used including the configuration (scale, environment
+                    variables etc)
+                </li>
+                <li>
+                    The postgres database - hackathon schemas, users and the configuration allowing the hackathon
+                    service to access it (bindings)
+                </li>
+                <li>
+                    The S3 service - hackathon buckets data, and the configuration allowing the hackathon service to
+                    access it (bindings)
+                </li>
+            </ul>
+            <br/>
+            <p>
+                The ability to both create copies and later restore individual components falls under the responsibility
+                of the "Ocopea Brokers".
+
+            </p>
+
+            <br/>
+            <br/>
+
+            <div class="title">Ocopea Brokers</div>
+
+            <p>
+                Ocopea brokers are the components that allow Ocopea to interact with the world.
+                Deployment of containers, provisioning and creation of infrastructure services copies is
+                all delegated to brokers.
+                <br/>
+                Brokers are managed by the ocopea component called the <span style="font-weight: bold">Site</span>.
+                Every Ocopea site is responsible for a local geographical location - either a public cloud availability zone or an on premises datacenter.
+                Sites can be aggregated by a central component called the Ocopea <span style="font-weight: bold">Hub</span>.
+
+                <br/>
+                There are three types of brokers:
+            </p>
+            <br/>
+            <ul class="list">
+                <li>
+                    PSB - PaaS/Container Orchestrator brokers
+                </li>
+                <li>
+                    DSB - Data service brokers
+                </li>
+                <li>
+                    CRB - Copy repository brokers
+                </li>
+            </ul>
+
+            <br/>
+            <div class="image-holder">
+                <img src="assets/ocopea-components.png"
+                     width="1024px"
+                     alt="Ocopea high level components"
+                />
+            </div>
+
+            <br/>
+            <br/>
+            <div class="title">PSB</div>
+            <p>
+                The PSB is responsible for communicating with the PaaS or the container orchestrator. It is used by
+                the Ocopea site to deploy the stateless containers and bind them to infrastructure services
+                (Data Services)
```

Comment:
Is Data Service actually a DB say in hackathon case?

Reply:
DB is a kind of data service...

Reply:
Ok thanks. If Mike is okay with your changes will push in then.